Xamarin.Forms.Platform.UAP/FormsTextBox.cs

```diff
@@ -53,10 +57,12 @@ public class FormsTextBox : TextBox
 		public FormsTextBox()
 		{
 			TextChanged += OnTextChanged;
+			TextChanging += OnTextChanging;
```

Comment:
I haven't taken a deep look at `FormsTextBox`, but it looks like none of these events are being unsubscribed. Hmm.